{{cookiecutter.module_name}}/Makefile

```diff
@@ -100,18 +100,25 @@ json/format: | guard/program/jq
 	$(FIND_JSON) | $(XARGS) bash -c 'echo "$$(jq --indent 4 -S . "{}")" > "{}"'
 	@ echo "[$@]: Successfully formatted JSON files!"
 
-docs/%: README_PARTS := _docs/MAIN.md <(echo) <(terraform-docs markdown table .)
-docs/%: README_FILE ?= README.md
+SCRIPTS_DIR ?= scripts
```

Comment:
This creates a tight coupling linkage between this project and the tfdocs-awk project... They both need to know where the file is...

We actually already have a `BIN_DIR` in the PATH, and these are executables... Can we follow the pattern elsewhere in the Makefile to put executables in the `BIN_DIR`, then in `autodoc.sh` just run `terraform-docs.sh` rather than specify the path to the script?

Reply:
updated